pkg/3scale/amp/auto-generated-templates/amp/amp-eval-s3.yml

```diff
@@ -789,6 +789,10 @@ objects:
               secretKeyRef:
                 key: password
                 name: backend-internal-api
+          - name: CONFIG_LISTENER_PROMETHEUS_METRICS_PORT
```

Comment:
I think we should add this to templates upgrade procedure. New 3scale installations through templates will have this environment variables so we should be consistent between existing and new templates installation.

Reply:
Indeed, this has to be included in the templa upgrade procedure